app/build.gradle

```diff
@@ -229,7 +229,8 @@ dependencies {
 
     implementation Dependencies.google_material
 
-    implementation 'io.sentry:sentry-android:5.6.2'
+    implementation Dependencies.sentry
+    implementation "org.mozilla.components:lib-crash-sentry:${AndroidComponents.VERSION}"
```

Comment:
I think we already have this dependency a bit below on line 268 https://github.com/mozilla-mobile/focus-android/blob/ea8bfa5ab9330c4bac6f49991535228f07477e39/app/build.gradle#L268

Reply:
yes sorry . I will refert

Reply:
No worries :) 
To keep the AC in sync, I also update the version that we compile again and added you as reviewer https://github.com/mozilla-mobile/android-components/pull/12197 :)